transports/sftp/src/main/java/org/mule/transport/sftp/SftpMessageReceiver.java

```diff
@@ -223,6 +223,7 @@ public MuleEvent process() throws Exception
             {
                 sftpStream.setErrorOccurred();
             }
+            throw e;
```

Comment:
really? I understand that it's "common sense" for this exception to be thrown here but I'm surprised that it isn't and still works since this transport is used a lot.

Are you sure about the need/impact of doing this?